packages/fxa-auth-server/lib/firestore-db.ts

```diff
@@ -15,8 +15,10 @@ export function setupFirestore(config: ConfigType) {
     delete fsConfig.credentials;
   }
 
+  const testing = !(fsConfig.keyFilename || fsConfig.credentials);
+
   // Utilize the local firestore emulator when the env indicates
-  if (process.env.FIRESTORE_EMULATOR_HOST) {
+  if (process.env.FIRESTORE_EMULATOR_HOST || testing) {
```

Comment:
Why isn't process.env.FIRESTORE_EMULATOR_HOST enough? It's defined when I run the local tests. Also `fsConfig.keyFilename` and `fsConfig.credentials` are both truthy based on the default config.

Reply:
Discussed this sync; my debugger was attached to the auth server process, not the Mocha test process. In the test context, the env var is `undefined`, so that's where `testing` kicks in which allows us to use the Firestore emulator in the test (the `mockConfig.authFirestore` in the test omits `keyFilename` and `credentials`).